deployments/ooi/image/binder/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Note that there must be a tag
-FROM pangeo/pangeo-notebook:afe4262
+FROM pangeo/pangeo-notebook:2020.07.28
```

Comment:
this should be `pangeo/base-image:2020.07.28` if you want to manage your own environment.yml etc